faq/index.html

```diff
@@ -25,86 +45,95 @@
     <link href="../assets/css/faces.css" rel="stylesheet">
 
     <!-- Just for debugging purposes. Don't actually copy these 2 lines! -->
-    <!--[if lt IE 9]><script src="../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
+    <!--[if lt IE 9]>
+    <script src="../assets/js/ie8-responsive-file-warning.js"></script><![endif]-->
     <script src="../assets/js/ie-emulation-modes-warning.js"></script>
 
     <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
     <!--[if lt IE 9]>
-      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
-      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
+    <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
+    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
 
-    <link href="../assets/css/flag-icon.min.css" rel="stylesheet" />
+    <link href="../assets/css/flag-icon.min.css" rel="stylesheet"/>
 
     <script language="javascript">
-      window.route = { section: 'community', page: 'organizers' }
+        window.route = {section: 'community', page: 'organizers'}
     </script>
-  </head>
+    <script>
+        (function (i, s, o, g, r, a, m) {
+            i['GoogleAnalyticsObject'] = r;
+            i[r] = i[r] || function () {
+                    (i[r].q = i[r].q || []).push(arguments)
+                }, i[r].l = 1 * new Date();
+            a = s.createElement(o),
+                m = s.getElementsByTagName(o)[0];
+            a.async = 1;
+            a.src = g;
+            m.parentNode.insertBefore(a, m)
+        })(window, document, 'script', '../assets/js/analytics.js', 'ga');
+
+        ga('create', 'UA-80867706-1', 'auto');
+        ga('send', 'pageview');
 
-  <body>
-    <!-- Fixed navbar -->
-    <nav class="navbar navbar-default navbar-fixed-top navbar-inverse text-uppercase" ng-app="scipyla.mainnav" ng-controller="MainNavCtl">
-      <ng-include src="'../assets/views/menu_ppal.html'"></ng-include>
-    </nav>
+    </script>
+</head>
 
-    <div class="container" ng-app="scipyla.md" id="container">
+<body>
+<!-- Fixed navbar -->
+<nav class="navbar navbar-default navbar-fixed-top navbar-inverse text-uppercase" ng-app="scipyla.mainnav"
+     ng-controller="MainNavCtl">
+    <ng-include src="'../assets/views/menu_ppal.html'"></ng-include>
+</nav>
 
-      <div class="row">
+<div class="container" ng-app="scipyla.md" id="container">
+
+    <div class="row">
         <div class="col-xs-12">
-          <div markdown-it ng-include="'../docs/pt/faq.pt.md'" class="md">
-          </div>
+            <div markdown-it ng-include="'/docs/pt/faq.pt.md'" class="md">
```

Comment:
I believe it was working with relative path.

Reply:
yes